WNPRC_EHR/src/typings/main.ts

```diff
@@ -5,4 +5,37 @@ export interface ConfigProps {
     sort?: string;
     containerPath?: string;
     filterArray?: Array<any>;
-}
\ No newline at end of file
+}
```

Comment:
Is this the animal abstract for the weight form?

Reply:
The AnimalInfoPane.tsx is the animal abstract for the weight form yes. It is different from the one used in the animal history.